zerver/lib/outgoing_webhook.py

```diff
@@ -205,6 +211,39 @@ def send_response_message(
     )
 
 
+def add_response_reaction(
+    bot_id: int, message_info: Dict[str, Any], reaction_data: Dict[str, Any]
+) -> None:
+    """
+    This function is similar to send_response_reaction, but it adds reaction instead.
+
+    bot_id is the user_id of the bot sending the response
+
+    message_info is used to address the message to be added with the reaction and
+    should have at least these fields:
```

Comment:
Better to phrase this as "message_info contains information that identifies the message to add the reaction to. This parameter should have at least the following fields:"

Reply:
Or is this the message that one adds in reply along with the reactions to the previous message?

Reply:
Good advice, there is indeed some confusion here. Just changed the docstring.